crates/cli/src/validate.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+use sightglass_artifact::WasmBenchmark;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// Check that a Wasm benchmark is runnable in this tool.
+#[derive(StructOpt, Debug)]
+#[structopt(name = "validate")]
+pub struct ValidateCommand {
+    /// The path to the WebAssembly benchmark module; this file should import `bench.start` and
+    /// `bench.end`.
+    #[structopt(
+        index = 1,
+        required = true,
+        value_name = "WASMFILE",
+        parse(from_os_str)
+    )]
+    benchmark: PathBuf,
+}
+
+impl ValidateCommand {
+    pub fn execute(&self) -> Result<()> {
+        WasmBenchmark::from(&self.benchmark).is_valid()?;
+        Ok(())
```

Comment:
It might be good to retain some way to print valid/invalid output for scripts that may validate multiple files. I'm fine with it being turned off by default but maybe a log::info here or in the lower levels?